src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/_params.py

```diff
@@ -1086,7 +1093,8 @@ def _configure_security_policy_storage_params(arg_ctx):
                 'virtual_network_subnet_id',
                 'vcores',
                 'storage_size_in_gb',
-                'collation'
+                'collation',
+                'proxy_override',
```

Comment:
If you add `public_data_endpoint_enabled` here, then you should not need to explicitly add it in `custom.py`

Reply:
Done